pkg/test/runner/runner.go

```diff
@@ -117,7 +117,11 @@ func (r *Runner) runFnEval() error {
 		kptArgs = append(kptArgs, "--exec-path", r.testCase.Config.EvalConfig.ExecPath)
 	}
 	if r.testCase.Config.EvalConfig.FnConfig != "" {
-		kptArgs = append(kptArgs, "--fn-config", r.testCase.Config.EvalConfig.FnConfig)
+		fnConfigPath, err := filepath.Abs(filepath.Join(r.testCase.Path, r.testCase.Config.EvalConfig.FnConfig))
```

Comment:
r.testCase.Config.EvalConfig.FnConfig is slash separated. Where is it converted to OS path?